app/templating/summary/question.py

```diff
@@ -74,16 +87,20 @@ def _find_other_value_in_answers(self, answer_store, answer_schema, answer, grou
     def _build_answer(self, answer_store, question_schema, answer_schema, answer_schemas, group_instance, answer_value=None):
         if answer_value is None:
             return None
+
         if question_schema['type'] == 'DateRange':
             return self._build_date_range_answer(answer_store, answer_value, answer_schemas, group_instance)
-        if answer_schema['type'] in ['Checkbox', 'MutuallyExclusiveCheckbox']:
-            checkbox_answers = self._build_checkbox_answers(answer_value, answer_schema)
-            return checkbox_answers or None
-        if answer_schema['type'] == 'Radio':
-            return self._build_radio_answer(answer_value, answer_schema)
-        if answer_schema['type'] == 'Dropdown':
-            return self._build_dropdown_answer(answer_value, answer_schema)
-        return answer_value
+
+        answer_builder = {
+            'Checkbox': self._build_checkbox_answers,
+            'MutuallyExclusiveCheckbox': self._build_checkbox_answers,
+            'Radio': self._build_radio_answer,
+            'Dropdown': self._build_dropdown_answer,
+        }
+
+        return answer_builder[answer_schema['type']](answer_value, answer_schema) \
```

Comment:
I think this should be rewritten to 
```
if answer_schema['type'] in answer_builder.keys():
	return answer_builder[answer_schema['type']](answer_value, answer_schema)
else:
	return answer_value
```
I've never been a fan of writing long 'single' line if else's

Reply:
Agreed, I remembered at lunch that the else is superfluous :)